vsintegration/src/.gitignore

```diff
@@ -0,0 +1 @@
+TestResults
```

Comment:
I did not realize anyone was running the unit tests through VS test runner. Is there some config that can be added to the project so that logs go in the normal place under %root%\tests\TestResults?